pytest_catchlog.py

```diff
@@ -122,6 +201,11 @@ def __init__(self, config):
         self.formatter = logging.Formatter(
                 get_option_ini(config, 'log_format'),
                 get_option_ini(config, 'log_date_format'))
+        self.handler = logging.StreamHandler(sys.stderr)
+        self.handler.setFormatter(self.formatter)
+        # The root logging should have the lowest logging level to allow all
+        # messages to be "passed" to the handlers
+        logging.root.setLevel(logging.NOTSET)
```

Comment:
For the specific handler and logger, right? Don't know if we'll get 3rd party logging without it.... I'll try.